DOCUMENTATION.md

```diff
@@ -0,0 +1,137 @@
+- File difference tracker with syntax highlighting for the [supported languages](LANGUAGES.md).
+- Commit log viewer
+- Basic git operations such as staging, un-staging, committing and pushing to remote repo.
+
+## How to use
+
+### Left Pane Menu
+
+- **Repositories** - To check tracked files changes, line based changes with syntax highlighting and git operations (staging, unstaging, commiting changes and pushing changed to remote)
+- **Settings** - To check and edit internal data file, to remove a repo from gitconvex and to update the active port.
+- **Help** - Includes documentation link and various options to report an issue or to submit feedback.
+
+### Adding a new repo
+- Use "+" at the bottom right corner to add a repo.
+![Picture1](https://user-images.githubusercontent.com/65342122/87232632-0eff7480-c3de-11ea-8a9f-f0a6cf9cd6ee.png)
+
+- Enter repo name and paste the repo path. If the folder is not a git repo then check the "*Check this if the folder is not a git repo*" checkbox to initialize git.
+![Picture2](https://user-images.githubusercontent.com/65342122/87232637-16268280-c3de-11ea-9f9d-708c5a3eb668.png)
+
+- The newly added repo will be displayed as a card in the dashboard
+![Picture17](https://user-images.githubusercontent.com/65342122/87243016-d13b3400-c44f-11ea-88ec-c4d14cbfbf97.png)
+
+### Repo Details
+- Click on the repo card to get the following details about the repo
+```
+- The list of branches
+- Commit logs
+- Latest commit
+- Active branch and available local branches
+- Remote repo URL and host
+- Files and folders tracked by git
+```
+The repo detail view also provides features for performing the following git operations,
+```
+- Adding a new branch
+- Pulling changes from remote
+- Fetching changes from remote
+- Adding a new remote repo
+```
+![Picture4](https://user-images.githubusercontent.com/65342122/87232642-1f175400-c3de-11ea-8ead-80cd5ab4c37c.png)
+
+![enter image description here](https://user-images.githubusercontent.com/65342122/87232671-71587500-c3de-11ea-8ff5-25fb95b08553.png)
+
+#### Add new branch
+![Picture5](https://user-images.githubusercontent.com/65342122/87232643-23437180-c3de-11ea-9d9e-7e3c3789c32e.png)
+
+Note: In a newly initialized git repo, the newly added branch will be considered by git only after a initial commit
+
+## Repositoru Menu
+- The **Repository** menu has three sections : File view, Git difference and Git operation, which will be displayed based on the selected repo. 
+
+### File View
+- File view shows the New/Untracked/Modified/Deleted files.
+![Picture6](https://user-images.githubusercontent.com/65342122/87232644-29d1e900-c3de-11ea-9adc-03fb4e690882.png)
+
+### Git Operations
+- Git operations module consists of three basic git operations( Stage all changes, commit changes, push to remote ). Below this file status is display and the files can be staged with add action or click on "stage all changes" to stage all the files from the chosen repo.
```

Comment:
change "consists of..." to, "lets you handle...'

Also, change the "Below this file status..." to "Below this option, the files will be displayed..."